changelogs/unreleased/6586-document-in-and-not-in.yml

```
@@ -0,0 +1,5 @@
+description: Add documentation for 'in' and 'not in' operators
+issue-nr: 6586
+change-type: patch
+destination-branches: [master]
```

> The "not in" operator is new right? it's not present in previous versions I mean?

It is available on iso6

```suggestion
destination-branches: [master, iso6]
```